python/cudf/cudf/tests/test_timedelta.py

```diff
@@ -448,43 +450,60 @@ def test_timedelta_dataframe_ops(df, op):
         ),
     ],
 )
-def test_timedelta_series_ops_with_scalars(data, other_scalars, dtype, op):
+@pytest.mark.parametrize("use_cudf_scalar", [False, True])
+def test_timedelta_series_ops_with_scalars(
+    data, other_scalars, dtype, op, use_cudf_scalar
+):
+    if (
+        isinstance(other_scalars, np.timedelta64)
+        and other_scalars.dtype not in dtypeutils.TIMEDELTA_TYPES
+        and use_cudf_scalar
+    ):
+        pytest.skip()
```

Comment:
Instead of skipping bad combinations we should rework the combinations in the parametrization

Reply:
Separated these into their own test